slam-libraries/Makefile

```diff
@@ -85,10 +85,14 @@ else
 	cd viam-cartographer && ./scripts/build_cartographer.sh && ./scripts/build_viam_cartographer.sh
 endif
 
+install-lua-carto:
+	sudo mkdir -p /usr/local/share/cartographer/lua_files/
+	sudo cp viam-cartographer/lua_files/* /usr/local/share/cartographer/lua_files/
+	sudo cp viam-cartographer/cartographer/configuration_files/* /usr/local/share/cartographer/lua_files/
 
 testcarto:
 	cd viam-cartographer && ./scripts/test_cartographer.sh
 
-carto-all: bufinstall buf setupcarto buildcarto testcarto
+carto-all: bufinstall buf setupcarto buildcarto install-lua-carto testcarto
```

Comment:
[opt] I think install-lua-carto should be a part of the setup or build makes.

Reply:
wanted it separate from setup just because its a unique dependency that installs locally rather than using a package manager, and we do not want our build steps to install things, users should actively choose to install the dependency.